tox.ini

```
@@ -18,6 +18,7 @@ deps=
     pytest-mock
     pytest-raises
     jsonschema
+    osqp>=0.6.0
```

Assuming semantic versioning, I'd prefer to constrain it to < 1.0 as well.

```suggestion
    osqp ~=0.6
```